server/main.lua

```diff
@@ -142,31 +142,15 @@ QBCore.Functions.CreateCallback('prison:server:checkThrowable', function(source,
     local Player = QBCore.Functions.GetPlayer(source)
 
     if not Player then return cb(false) end
-
-    if QBCore.Shared.Weapons[weapon]["name"] == "weapon_snowball" then
-        Player.Functions.RemoveItem("weapon_snowball", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_pipebomb" then
-        Player.Functions.RemoveItem("weapon_pipebomb", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_molotov" then
-        Player.Functions.RemoveItem("weapon_molotov", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_stickybomb" then
-        Player.Functions.RemoveItem("weapon_stickybomb", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_grenade" then
-        Player.Functions.RemoveItem("weapon_grenade", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_bzgas" then
-        Player.Functions.RemoveItem("weapon_bzgas", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_proxmine" then
-        Player.Functions.RemoveItem("weapon_proxmine", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_ball" then
-        Player.Functions.RemoveItem("weapon_ball", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_smokegrenade" then
-        Player.Functions.RemoveItem("weapon_smokegrenade", 1)
-    elseif QBCore.Shared.Weapons[weapon]["name"] == "weapon_flare" then
-        Player.Functions.RemoveItem("weapon_flare", 1)
-    else
-        return cb(false)
+    local throwable = false
+    for _,v in pairs(Config.Throwables) do
+        if QBCore.Shared.Weapons[weapon].name == v then
```

Comment:
Appending `weapon_` back in could look something like this?
`if QBCore.Shared.Weapons[weapon].name == 'weapon_' .. v then`